### PR TITLE
add reorg page

### PR DESCRIPTION
**Pull request by SvenMuller95**
noting, for some reason it included changes to hyperrpc URL page (basically changes that were made by someone else previously). I didn't touch these (checkout out main, git pull, created my branch, etc.) but in any case, it's all correct and as intended anyways :)

### CODE REVIEW

File: docs/HyperIndex/Advanced/reorgs-support.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+:::note
+This feature, while functional, is not optimized. If you choose to have this feature enabled, please be aware that historical indexing may slow down. We want to ensure blazing-fast indexing and are currently working on improving this. 
+:::
+
+Chain reorganizations are handled automatically by HyperIndex.
+
+To turn this feature on, you can set the ```rollback_on_reorg``` flag in your config.yaml to ```true```:
+```yaml
+rollback_on_reorg: true
```
Comment (moose-code) [inline docs/HyperIndex/Advanced/reorgs-support.md:16]: Its already on by default, you don't need to set it. 

You only need to set it to "off" is you don't want it

**Review by moose-code — CHANGES_REQUESTED** [review]
Nice!

File: docs/HyperIndex/Advanced/reorgs-support.md (hunk 1 above)
Comment (SvenMuller95) [reply docs/HyperIndex/Advanced/reorgs-support.md:16]: kind adisagree with latest version it being off by default

**Review by moose-code — APPROVED** [review]
LGTM 💪

**Pull request merged**